src/accelerate/accelerator.py

```diff
@@ -3279,3 +3291,16 @@ def verify_device_map(self, model: torch.nn.Module) -> bool:
                 return True
 
         return False
+
+    def _lomo_backward(self, loss: torch.Tensor, learning_rate: float) -> None:
```

Comment:
This should be in `.backward()`, where we can check for `learning_rate` as part of `**kwargs` IMO